h/static/scripts/directive/form-input.coffee

```diff
@@ -0,0 +1,33 @@
+module.exports = ->
+  link: (scope, elem, attr, [model, validator]) ->
+    return unless  model
```

Comment:
This has also changed. The changes here look fine to me but I would really suggest you resist the urge to fix all the things at once in future. It's *so* much easier to review the pieces where the only thing that's changed is `module.exports` at the top and `angular.module` at the bottom.